include/robot_interfaces/cereal_eigen.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file
+ * @brief Serialization functions for serializing Eigen matrices and arrays
+ *        with cereal.
+ * @authors Azoth, eudoxos
+ * @date 2020-01-15
+ * @license CC BY-SA 4.0
+ * @todo Move this to some "serialization tools" package.
+ *
+ * Taken from https://stackoverflow.com/a/51944389/2095383.
+ */
+#include <Eigen/Eigen>
+#include <type_traits>
+
+namespace cereal
+{
+template <class Archive, class Derived>
+inline typename std::enable_if<
+    traits::is_output_serializable<BinaryData<typename Derived::Scalar>,
+                                   Archive>::value,
+    void>::type
+save(Archive& ar, Eigen::PlainObjectBase<Derived> const& m)
```

Comment:
variable names could be a bit more descriptive

Reply:
done